README.md

```diff
@@ -654,6 +654,74 @@ instance.delete()
 
 **Do not use delete() in production!**
 
+Bulk-updating sandboxes
+-----------------------
+
+OpenCraft IM provides a tagging mechanism that you can use to update instances in bulk.
+The following script shows how to update multiple instances at once from the Django shell:
+
+```python
+# Author: Brandon DeRosier <bdero@opencraft.com>
```

Comment:
@antoviaque How do you envision that working for things like instance upgrades (e.g. ficus.2 to ficus.3), settings changes (e.g. enable CREATOR_GROUP for all instances), resource sharding (e.g. moving 1/2 of our instances to a new RabbitMQ server), etc.? There's no pull request involved for those things so I don't see it as "continuous integration".